README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@damithc 
> As I mentioned above, I'm open for a better name that doesn't make sweeping claims but also not NUS specific.

Maybe this could be part of SE-EDU? Since besides documenting what TEAMMATES, SE-EDU, PowerPointLabs already do, it also serves to *educate* students/developers who would like to contribute/set up an OSS project? After all, some of these stuff (e.g. coding standards) are applicable to both OSS and non-OSS projects.